src/dev/konnector.md

```diff
@@ -2,110 +2,376 @@
+    }
+  },
+  'article'
+)
+```
+
+This code will loop on `<article />` and for each item will create a JSON object with the selector `sel` and the value of attribute `attr` if specified, otherwise it takes the value of the child node, this value can be edited with the `parse` function.
+Here is a sample for the following markup from http://books.toscrape.com:
+
+```html
+<article class="product_pod">
+  <div class="image_container">
+    <a href="catalogue/a-light-in-the-attic_1000/index.html">
+      <img src="media/cache/2c/da/2cdad67c44b002e7ead0cc35693c0e8b.jpg" alt="A Light in the Attic" class="thumbnail">
+    </a>
+  </div>
+  <p class="star-rating Three">
+    <i class="icon-star"></i>
+    <i class="icon-star"></i>
+    <i class="icon-star"></i>
+    <i class="icon-star"></i>
+    <i class="icon-star"></i>
+  </p>
+  <h3>
+    <a href="catalogue/a-light-in-the-attic_1000/index.html" title="A Light in the Attic">A Light in the ...</a>
+  </h3>
+  <div class="product_price">
+    <p class="price_color">£51.77</p>
+    <p class="instock availability">
+      <i class="icon-ok"></i>
+      In stock
+    </p>
+    <form>
+      <button type="submit" class="btn btn-primary btn-block" data-loading-text="Adding...">Add to basket</button>
+    </form>
+  </div>
+</article>
+```
+
+And we will get the following JSON object:
+
+```json
+{
+  "title": "A Light in the Attic",
+  "amount": 51.77,
+  "url": "http://books.toscrape.com/catalogue/a-light-in-the-attic_1000/index.html",
+  "fileurl": "http://books.toscrape.com/media/cache/2c/da/2cdad67c44b002e7ead0cc35693c0e8b.jpg",
+  "filename": "A Light in the Attic.jpg"
+}
+```
+
+The code sample includes some other function to manipulate the result object, but we have the idea.
+Once we build a correct object, we can save it to Cozy Stack.
+
+#### Save data to Cozy Stack
+
+In the example we use some built-in function to save a bill to the Cozy Stack.
+But there is a bunch of functions available depending on what you want:
+
+- [`addData`](https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/docs/api.md#adddata)
+- [`filterData`](https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/docs/api.md#module_filterData)
+- [`saveBills`](https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/docs/api.md#module_saveBills)
+- [`saveFiles`](https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/docs/api.md#module_saveFiles)
+- and so on…
+
+We can find more information in the [libs repository](https://github.com/konnectors/libs).
+
+__Now we pass on every steps, it is time to test the connector with `yarn standalone`.__
+We will see in the following how to connect it effectively to a Cozy Stack.
 
 ### Connector structure
 
-Basically, a connector is just a function passed to the BaseKonnector constructor, and which
+Basically, a connector is just a function passed to the `BaseKonnector` constructor, and which
```

Comment:
Everything from here on seems either very low level, or has already been covered I think?

Reply:
Agreed, but this is still interesting. Maybe we could split (now very long) page in two sections : A "your first connector" part and a "technical explanations" part?

But this can be done in another PR.